packages/babel-parser/test/fixtures/es2022/class-private-methods/async-generator/options.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
This commit is created from https://gist.github.com/JLHwung/970c52d63fd3b5ee14e3ada1cd227d33